lms/templates/api/canvas/oauth2_redirect_error.html.jinja2

```diff
@@ -6,39 +6,41 @@
     <div class="Dialog__content LMSFilePicker__dialog">
       {% if invalid_scope %}
         <h1 class="Dialog__title">Developer Key Scopes Missing</h1>
+        <div class="ErrorDisplay">
+          <p>
+            A Canvas admin needs to edit Hypothesis's developer key and add these
+            scopes:
+          </p>
 
-        <p>
-          A Canvas admin needs to edit Hypothesis's developer key and add these
-          scopes:
-        </p>
+          <ol>
+            {% for scope in scopes %}
+              <li><code>{{ scope }}</code></li>
+            {% endfor %}
+          </ol>
 
-        <ol>
-          {% for scope in scopes %}
-            <li><code>{{ scope }}</code></li>
-          {% endfor %}
-        </ol>
-
-        <p>For more information see:
-        <a target="_blank" rel="noopener noreferrer" href="https://github.com/hypothesis/lms/wiki/Canvas-API-Endpoints-Used-by-the-Hypothesis-LMS-App">Canvas API Endpoints Used 
-y the Hypothesis LMS App</a>.
-        </p>
-      {% else %}
-        <h1 class="Dialog__title">Authorization Failed</h1>
+          <p>For more information see:
+          <a target="_blank" rel="noopener noreferrer" href="https://github.com/hypothesis/lms/wiki/Canvas-API-Endpoints-Used-by-the-Hypothesis-LMS-App">Canvas API Endpoints Used 
+  y the Hypothesis LMS App</a>.
```

Comment:
There was an existing typo here ("y the Hypothesis LMS app" should be "by the Hypothesis LMS app").